### PR TITLE
chore: Markdown fixes for inputs/[a-m]*

### PR DESCRIPTION
**Pull request by srebhan**
- [x] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

This PR fixes markdown (and sample.conf) files for the input plugins starting with A to M. It also fixes the `sample.conf` embedding into the README for `cloudwatch_metric_streams`.

With this PR the mentioned input streams will pass `markdownlint` with the following configuration:

```yaml
{
    "MD013": {
      code_blocks: false,
      tables: false,
      headers: false
    },
    "MD033": {
      "allowed_elements": ["br"]
    }
}
```

### CODE REVIEW

**Review by powersj — COMMENTED** [review]
Thank you this will make PRs go a lot smoother.

**Review by powersj — COMMENTED** [review]
There were three linter items that are probably worth fixing:

```
plugins/inputs/jolokia2/jolokia2_proxy/README.md:1: 'Jolokia2 Agent plugin' does not match regexp ' Plugin$'
plugins/inputs/jolokia2/jolokia2_agent/README.md:1: 'Jolokia2 Proxy input plugin' does not match regexp ' Plugin$'
plugins/inputs/cloudwatch/README.md:135: long line in paragraph
```

**Comment by srebhan** [comment]
Will look those tomorrow. Only looked at the markdownlinter ones, but it make sense to also fix the other linter issues. Thanks for catching this!

**Comment by srebhan** [comment]
@powersj I fixed most of the `jolokia2` issues, but there is one remaining due to the fact that the plugin structure is different from anything else. The point of the linter is that the `Configuration` section should be followed by a `toml` code block, but this plugin requires some more explaination due to the `agent`/`proxy` split. Any idea on how to handle this?

**Comment by powersj** [comment]
Hmm is it possible to include both sample configs one right after another? And then below that include the explanations? That doesn't seem ideal though.

File: plugins/inputs/jolokia2/README.md [hunk 1]
```diff
@@ -205,3 +205,7 @@ configurations that apply to every `metric` declaration.
 
 Please help improve this list and contribute new configuration files by opening
 an issue or pull request.
+
+## Metrics
+
+## Example Output
```
Comment (reimda) [inline plugins/inputs/jolokia2/README.md:211]: Empty sections don't do anything but make the readme linter happy. I'd rather leave them out and ignore the readme linter errors until we have content for them.

File: plugins/inputs/jolokia2/jolokia2_agent/README.md [hunk 2]
```diff
@@ -29,3 +31,7 @@ The `jolokia2_proxy` input plugin reads JMX metrics from one or more _targets_ b
     mbean = "java.lang:type=Runtime"
     paths = ["Uptime"]
 ```
+
+## Metrics
+
+## Example Output
```
Comment (reimda) [inline plugins/inputs/jolokia2/jolokia2_agent/README.md:37]: empty sections

File: plugins/inputs/jolokia2/jolokia2_proxy/README.md [hunk 3]
```diff
@@ -37,3 +38,7 @@ The `jolokia2_agent` input plugin reads JMX metrics from one or more [Jolokia ag
     mbean = "java.lang:type=Runtime"
     paths = ["Uptime"]
 ```
+
+## Metrics
+
+## Example Output
```
Comment (reimda) [inline plugins/inputs/jolokia2/jolokia2_proxy/README.md:44]: empty sections

File: plugins/inputs/jolokia2/jolokia2_agent/README.md [hunk 4]
```diff
@@ -1,10 +1,12 @@
-# Jolokia2 Proxy input plugin
+# Jolokia2 Proxy Input Plugin
```
Comment (reimda) [inline plugins/inputs/jolokia2/jolokia2_agent/README.md:1]: This is the jolokia2 agent readme file but it's talking about the jolokai2 proxy plugin. Not sure how that happened.

File: plugins/inputs/jolokia2/jolokia2_proxy/README.md [hunk 5]
```diff
@@ -1,10 +1,11 @@
-# Jolokia2 Agent plugin
+# Jolokia2 Agent Input Plugin
```
Comment (reimda) [inline plugins/inputs/jolokia2/jolokia2_proxy/README.md:1]: Similar to above, this is the proxy readme talking about the agent plugin

**Review by reimda — COMMENTED** [review]
We talked about enabling the markdownlint line length checker and disabling the one in tools/readme_linter. Did you change the linter configuration locally for this PR? Do you want to include that change?

File: plugins/inputs/clickhouse/README.md [hunk 6]
```diff
@@ -13,34 +13,32 @@ This plugin gathers the statistic data from
   username = "default"
 
   ## Password for authorization on ClickHouse server
-  ## example: password = "super_secret"
+  # password = "super_secret"
```
[the remark this answers is not in the document]
Comment (srebhan) [reply plugins/inputs/clickhouse/README.md:16]: Well this wasn't meant for a fixup of the README styles but only about linter fixes... But I can change this anyway.

File: plugins/inputs/jolokia2/README.md (hunk 1 above)
Comment (srebhan) [reply plugins/inputs/jolokia2/README.md:211]: I agree. Will undo the change.

File: plugins/inputs/jolokia2/jolokia2_agent/README.md (hunk 4 above)
Comment (srebhan) [reply plugins/inputs/jolokia2/jolokia2_agent/README.md:1]: Well the main issue is the non-standard structure of the Jolokia2 plugin(s). If you agree I will (in a separate PR) move `plugins/inputs/jolokia2/common` to `plugins/common/jolokia2` and flatten the structure to make `_agent` and `_proxy` visible under `plugins/inputs`... What do you think?

Comment (reimda) [reply plugins/inputs/jolokia2/jolokia2_agent/README.md:1]: That sounds good. Having them both directly in plugins/inputs will be much better. Thanks

**Review by reimda — APPROVED** [review]

**Review by powersj — APPROVED** [review]

**Pull request merged**